src/openapi.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  createPreset,
+  semanticReleaseGit,
+  semanticReleaseOpenApi,
+} from "~/_config";
+
+/**
+ * Semantic release configuration preset for OpenAPI projects. It adds the gradle-semantic-release-plugin,
+ * the semantic-release-openapi plugin, and ensures that the gradle.properties file and swagger spec gets committed
+ */
+export = createPreset([
+  "gradle-semantic-release-plugin",
```

Comment:
Should the openapi plugin know about gradle? Ideally no, but since `semantic-release` doesn't allow to combine multiple presets and we use open api with gradle. I would add a comment here explaining that. 

If this was just an open api plugin, I would think that the gradle plugin wouldn't be needed